test/customization_test.rb

```diff
@@ -155,6 +155,17 @@ def customization_build_output(printer)
     printer = Oppen::Wadler.new(width: 45, space: ->(*_args) { '*' })
     _ { printer.output }.must_raise ArgumentError
   end
+
+  it 'must only call space when indent is positive' do
+    printer = Oppen::Wadler.new(width: 10, space: ->(_n) { '*******' })
+    printer.text 'Hello'
+    printer.break
+    printer.text 'World'
+    _(printer.output).must_equal <<~LANG.chomp
+      Hello
+      World
+    LANG
```

Comment:
This doesn't seem to test positive indentation at a first glance. Could you explain how it does (in its doc comments).